modules/dev/src/tests/replDb.ts

```diff
@@ -0,0 +1,33 @@
+import { TestNamespace, TestObject } from "../types";
+import { replDb } from "@replit/extensions";
+import { assert } from "chai";
+
+const tests: TestObject = {
+  list: async (log) => {
+    await replDb.set({
+      key: "__extensions_test_key__",
+      value: "test value",
+    });
```

Comment:
again let's include a timestamp here


```suggestion
    let key = `__extensions_test_key__/${Date.now()}`
    await replDb.set({
      key,
      value: "test value",
    });
```